08-components-lab/src/01-timeconverter.jsx

```diff
@@ -8,5 +8,72 @@
 
 import React from 'react';
 
-export default React.createClass({
-});
+export default class TimeConverter extends React.Component {
+    static propTypes = {
+        initialHours: React.PropTypes.number,
+        initialMinutes: React.PropTypes.number,
+        initialSeconds: React.PropTypes.number,
+    };
+
+    static defaultProps = {
+        initialHours: 0,
+        initialMinutes: 0,
+        initialSeconds: 0,
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            hours: props.initialCount,
+            minutes: (props.initialCount ),
+            seconds: (props.initialCount)
+        };
+    }
+
+    handleHours = (event) => {
+        this.setState({
+            hours: event.target.value,
+            minutes: (event.target.value * 60),
+            seconds: (event.target.value * 60 * 60)
+        });
+    }
+
+    handleMinutes = (event) => {
+        this.setState({
+            hours: event.target.value/60,
+            minutes: (event.target.value),
+            seconds: (event.target.value * 60)
+        });
+    }
+
+    handleSeconds = (event) => {
+        this.setState({
+            hours: event.target.value/60/60,
+            minutes: (event.target.value /60),
+            seconds: (event.target.value)
+        });
+    }
+
+    render() {
+        var hoursValue = this.state.hours;
+        var minutesValue = this.state.minutes;
+        var secondsValue = this.state.seconds;
+        return (<div>
+            <p>
+                <br></br>
```

Comment:
I advice not to use `<br/>` tags. If you want proper separation between html elements better do it with css or use inline style with "style" prop.